src/models/character.py

```
@@ -1,19 +1,84 @@
+from ..display.common.message_view import MessageView
+from .base_types import GameEntity, EffectTrigger
+from .effects.base import BaseEffect
+from .items.equipment import Equipment
 from .character_classes import CharacterClass
-from .items import Item, Equipment
 from .status_effects import StatusEffect
+from ..config.settings import STARTING_INVENTORY
+from .inventory import get_starting_items
+from .items.sets import SetBonus, ITEM_SETS
+from .base_types import ItemType
+from .effects.base import BaseEffect  # Use BaseEffect instead of ItemEffect
```

Ruff (0.8.2)

Redefinition of unused `BaseEffect` from line 8

Remove definition: `BaseEffect`

(F811)